linera-core/src/worker.rs

```diff
@@ -525,8 +525,12 @@ where
         for value in blobs {
             self.cache_recent_value(value.clone());
         }
-        self.storage.write_values(blobs).await?;
-        self.storage.write_certificate(&certificate).await?;
+        let (result_blob, result_certificate) = tokio::join!(
```

Comment:
We may want to start using non-tokio equivalent for this kind of stuff.

Reply:
Ok, if you expain what you would like, I could do it.

Reply:
This one https://doc.rust-lang.org/std/future/macro.join.html but sadly it's still unstable.